feeds/wlan-ap/opennds/files/lib/login.sh

```diff
@@ -349,7 +349,7 @@ else
 
 	# You can also send a custom data string to BinAuth. Set the variable $custom to the desired value
 	# Max length 256 characters
-	custom="Custom data sent to BinAuth"
+	#custom="Custom data sent to BinAuth"
 	requested="$redirecturl"
```

Comment:
can we just delete the line if it is no longer needed ?

Reply:
sure, it's not needed for now